providers/base/bin/camera_quality_test.py

```diff
@@ -22,22 +22,51 @@
 import argparse
 from collections import deque
 import logging
+import os
 import time
 import sys
 
 import cv2
 import numpy as np
 
 from checkbox_support.vendor.brisque.brisque import BRISQUE
+from checkbox_support.helpers.timeout import timeout
 from tempfile import NamedTemporaryFile
 
+
 THRESHOLD = 60
 TIMEOUT = 10
 MIN_INTERVAL = 0.5
+PLAINBOX_SESSION_SHARE = os.environ.get("PLAINBOX_SESSION_SHARE", "")
```

Comment:
Instead of hardcoding the output location, use an argument (for instance `--output/-o`) when calling the script, so that it's easier to try out the script without calling Checkbox.